can/uwrt_mars_rover_can_2022.yaml

```diff
@@ -7,8 +7,694 @@ uwrt_mars_rover_can_bus:
   nodes:
   - GIMBAL
   - JETSON
+  - ODRIVE
   - PDB
   - SCIENCE
   roboteq_canids:
     0: TEMP
-  messages: null
+  messages:
+  - ODRIVE_heartbeatMessage_axis0:
+      comment: Heartbeat messages for odrive
+      id: 1
+      size: 8
+      senders:
+      - ODRIVE
+      receivers:
+      - JETSON
+      frequency: 0
+      signals:
+      - ODRIVE_axisError:
+          comment: Odrive axis errors
+          length: 32
+          is_signed: false
+          unit: ''
+          values:
+            0: ERROR_NONE
+            1: ERROR_INVALID_STATE
+            2048: ERROR_WATCHDOG_TIMER_EXPIRED
+            4096: ERROR_MIN_ENDSTOP_PRESSED
+            8192: ERROR_MAX_ENDSTOP_PRESSED
+            16384: ERROR_ESTOP_REQUESTED
+            131072: ERROR_HOMING_WITHOUT_ENDSTOP
+            262144: ERROR_OVER_TEMP
+            524288: ERROR_UNKNOWN_POSITION
+          scale: 1.0
+          offset: 0.0
+          min: 0.0
+          max: 1.0
+      - ODRIVE_axisCurrentState:
+          comment: Odrive axis current state
+          length: 8
+          is_signed: false
+          unit: ''
+          values:
+            0: AXIS_STATE_UNDEFINED
+            1: AXIS_STATE_IDLE
+            2: AXIS_STATE_STARTUP_SEQUENCE
+            3: AXIS_STATE_FULL_CALIBRATION_SEQUENCE
+            4: AXIS_STATE_MOTOR_CALIBRATION
+            6: AXIS_STATE_ENCODER_INDEX_SEARCH
+            7: AXIS_STATE_ENCODER_OFFSET_CALIBRATION
+            8: AXIS_STATE_CLOSED_LOOP_CONTROL
+            9: AXIS_STATE_LOCKIN_SPIN
+            10: AXIS_STATE_ENCODER_DIR_FIND
+            11: AXIS_STATE_HOMING
+            12: AXIS_STATE_ENCODER_HALL_POLARITY_CALIBRATION
+            13: AXIS_STATE_ENCODER_HALL_PHASE_CALIBRATION
+          scale: 1.0
+          offset: 0.0
+          min: 0.0
+          max: 1.0
+      - ODRIVE_controllerStatus:
+          comment: Odrive controller status
+          length: 8
+          is_signed: false
+          unit: ''
+          values:
+            0: CONTROL_MODE_VOLTAGE_CONTROL
+            1: CONTROL_MODE_TORQUE_CONTROL
+            2: CONTROL_MODE_VELOCITY_CONTROL
+            3: CONTROL_MODE_POSITION_CONTROL
+          scale: 1.0
+          offset: 0.0
+          min: 0.0
+          max: 1.0
```

Comment:
These lines don't seem right

Reply:
I wasn't too sure what to put here so these were temporary, do you know where I can find this info for Axis current state?

Reply:
These should only be used for signals that have ranges. This is an enum type? 

Although I'm just going off my knowledge of dbc. Idk if Cindy's script is expecting some yaml field here or smth

Reply:
You can only put in the scale and offset fields and the script will autogen the min/max fields. @rohanperi for enum-type signals, you shouldn't put min/max fields, just put in scale/offset and let the script autogen the rest